src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -112,9 +129,14 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISc
         private OnnxTransform(IHostEnvironment env, Arguments args, byte[] modelBytes = null) :
             base(Contracts.CheckRef(env, nameof(env)).Register(nameof(OnnxTransform)))
         {
+            //Contracts.CheckValue(env, nameof(env));
+            //Host = env.Register(RegistrationName);
```

Comment:
feel free to delete, Host now get created in base class (and sorry for this changes)